handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -67,6 +68,11 @@
     static final Set<String> SUPPORTED_PROTOCOLS_SET;
     static final String[] EXTRA_SUPPORTED_TLS_1_3_CIPHERS;
     static final String EXTRA_SUPPORTED_TLS_1_3_CIPHERS_STRING;
+    static final String[] NAMED_GROUPS;
+
+    // Use default that is supported in java 11 and earlier.
+    // See https://github.com/netty/netty-tcnative/issues/567
+    private static final String[] DEFAULT_NAMED_GROUPS = {  "P-256", "P-384", "X25519" };
```

Comment:
That is the naming that is used by BoringSSL and OpenSSL.

Reply:
Are these in order of preference? JDK has `x25519` as its "most preferred" group.

Reply:
Also, these names look different from the JDK names, which means this isn't a "compliant" implementation of that system property. People can't just switch SSL implementation without also modifying the names in this property, if they're using it.

Reply:
yeah maybe we should include some sort of a "mapper". That said the docs say that the names are not really "defined" and its mostly up to the implementation:

https://docs.oracle.com/en/java/javase/12/security/java-secure-socket-extension-jsse-reference-guide.html#GUID-9D7375F9-D688-436D-A214-02653F50ED32

Reply:
> Are these in order of preference? JDK has `x25519` as its "most preferred" group.

do you have some link to this ?

Reply:
Link: https://www.java.com/en/configure_crypto.html